typedb/common/exception.py

```diff
@@ -116,7 +116,7 @@ def __init__(self, code: int, message: str):
 NONEXISTENT_EXPLAINABLE_CONCEPT = ConceptErrorMessage(10, "The concept identified by '%s' is not explainable.")
 NONEXISTENT_EXPLAINABLE_OWNERSHIP = ConceptErrorMessage(11, "The ownership by owner '%s' of attribute '%s' is not explainable.")
 GET_HAS_WITH_MULTIPLE_FILTERS = ConceptErrorMessage(12, "Only one filter can be applied at a time to get_has. The possible filters are: [attribute_type, attribute_types, annotations]")
-
+HAS_TIMEZONE_INFO = ConceptErrorMessage(13, "A date-time attribute cannot accept timezone aware datetime objects.")
```

Comment:
I like the error message, but the variable name could be a bit more complete.
It doesn't tell me what has timezone info and why it's a problem